app/lib/features/chat/widgets/convo_list.dart

```diff
@@ -1,100 +1,99 @@
 import 'dart:ui';
-
-import 'package:acter/features/chat/models/joined_room/joined_room.dart';
 import 'package:acter/features/chat/providers/chat_providers.dart';
 import 'package:acter/features/chat/widgets/convo_card.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:implicitly_animated_reorderable_list_2/implicitly_animated_reorderable_list_2.dart';
 import 'package:implicitly_animated_reorderable_list_2/transitions.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 class ConvosList extends ConsumerWidget {
   const ConvosList({super.key});
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final chatList = ref.watch(chatListProvider);
-    final joinedRooms = ref.watch(joinedRoomListProvider);
-    var loaded = ref.watch(chatListProvider.select((e) => e.initialLoaded));
-    if (!loaded) {
-      return Center(
+    final chats = ref.watch(chatListProvider);
+    return chats.when(
+      initial: () => const SizedBox.shrink(),
+      loading: () => Center(
         heightFactor: 10,
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text('${AppLocalizations.of(context)!.loadingConvo}...'),
-            const SizedBox(height: 10),
-            const CircularProgressIndicator(),
-          ],
+        child: Text(
+          '${AppLocalizations.of(context)!.loadingConvo}...',
```

Comment:
shouldn't that `...` also be in there, if we already use it...?

```suggestion
          AppLocalizations.of(context)!.loadingConvo,
```

Reply:
Its not in there.